eval/nki_docs_parsing_scripts/nki_docs_parser.py

```diff
@@ -0,0 +1,116 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import time
+
+def copy_main_section_text(url, section_selector="section#nki-language-add", output_file="main_section.txt"):
```

Comment:
Is this the right entry point? I think there's a lot more of NKI content on the docs we'd also like to see here. Tutorials, ISA, Allocation, etc.